examples/common.c

```diff
@@ -391,3 +395,61 @@ int cred_acquire_cb(git_cred **out,
 	free(pubkey);
 	return error;
 }
+
+char *read_file(const char *path) {
+	FILE *file;
+	char *result;
+	size_t pos = 0;
+	size_t capacity = READ_SIZE;
+	size_t blocksRead= 0;
+	size_t totalRead = 0;
+
+	file = fopen(path, "r");
+
```

Comment:
Style: 2-line `if`s can lose their braces.

Reply:
Yes, that's my personal choice, but if without braces are all around the code so I will fix that